modules/microshift-greenboot-dir-structure.adoc

```diff
@@ -24,18 +24,23 @@ Greenboot redirects script and health check output to the system log. When you a
 
 Returning a nonzero exit code from any script means that script has failed. Greenboot restarts the system a few times to retry the scripts before attempting to roll back to the previous version.
 
-* `/etc/greenboot/check/required.d` contains the health checks that must not fail.
+* `/usr/lib/greenboot/check/required.d` contains the health checks that must not fail.
```

Comment:
The old version is still true, failing script in both `/usr/lib/lib/greenboot/check/required.d` and `/etc/greenboot/check/required.d` can trigger reboot